sonar/modules/validation/templates/validation/email/validated/fr.txt

```diff
@@ -0,0 +1,10 @@
+Hi {{ user.first_name }},
+
+Your record "{{ record }}" (#{{ record.pid.pid_value }}) has been approved by {{ moderator.first_name }} {{ moderator.last_name }} ({{ moderator.email }}).
+{% if comment %}
+{{ _('Moderator comment') }}:
+==================
+{{ comment }}
+==================
+{% endif %}
+The record will be published shortly, thank you for your submission!
```

Comment:
Translation seems missing. Check for other cases.

Reply:
I think we don't have to put translations here because we are in a template for a specific language.

Reply:
Ok then the text should be in french no?

Reply:
Ohhh yes I understand, but at this time I don't have the translations...